typings/custom-hash-node-map-defs.d.ts

```diff
@@ -0,0 +1,11 @@
+interface CustomHashNodeMap extends NamedNodeMap {
+  fill: {
+  value: string;
```

Comment:
Bad indentation. Again, proofread your changes please, otherwise we have needless extra back-and-forth.

Reply:
Done.